.github/workflows/ci.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     - uses: actions/checkout@v1
     - run: cargo build --verbose
     - run: |
-        cargo run &
+        RUST_LOG=info cargo run &
```

Comment:
Could you also add `RUST_BACKTRACE=1`?

Reply:
Of course! That is, Ionut, an excellent suggestion.